### ISSUE TITLE
Status messages missing ARIA role

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2347]
`.success` and `.error` status message containers are missing roles appropriate to programmatically determine the announcement of a status change (`role="alert"` or `role="status"`). See [SC 4.1.3](https://w3.org/TR/WCAG22#status-messages) for more.
